ConsentViewController/Classes/SPAction.swift

```diff
@@ -54,7 +54,13 @@ import Foundation
         return nil
     }
     public var pmPayload = SPJson()
-    public var publisherData: SPPublisherData = [:]
+    public var publisherData: [String: String] = [:] {
+        didSet {
+            encodablePubData = publisherData.mapValues { AnyEncodable($0) }
+        }
+    }
+    /// This property will be renamed to `publisherData` in the next major release
```

Comment:
I thought about it @Nevazhnovu . But it's a bit tricky since, in the future, we'll not exactly deprecate `publisherData`, but rather change its type (from `[String: String]` to `SPPublisherData` and remove `encodablePubData`.